packages/ec-linear/examples/hiff/main.rs

```diff
@@ -1,10 +1,10 @@
 // TODO: since inner attributes are unstable, we can't use rustversion here.
 // Once we revert this commit, this is proper again.
+#![allow(clippy::allow_attributes_without_reason)]
 #![allow(
-    clippy::allow_attributes_without_reason,
     clippy::arithmetic_side_effects,
-    // reason = "The tradeoff safety <> ease of writing arguably lies on the ease of writing side \
-    //           for example code."
+//     reason = "The tradeoff safety <> ease of writing arguably lies on the ease of writing side \
+//               for example code."
```

Comment:
Can we please not change this (yet) since it will make it way harder to revert everything once 1.81 lands - yes I know I will have to do a bunch of manual cleanup with allows() that were added later but I'd rather not make it more bothersome than it is already.

Reply:
And just for your understanding since you seemed not very clear about that on stream: this **will** come out with 1.81, which is the current nightly version - it was stabilized already (is not a #[feature()] gate anymore), in that version. The thing why it currently only works in nightly is that  nightly is on 1.81 already, whereas beta is on 1.80 and stable on 1.79 ( well till tomorrow, tomorrow nightly will move to 1.90, beta to 1.81 and stable to 1.80 ) - the release cycle is just that everything lands in nightly first and then moves through beta and to stable.

The only case where something gets `de-stabilized` (which is actually usually a pretty huge deal and happens almost never) is when a deal-breaking bug gets found during nightly or beta testing.

Reply:
Thanks – I didn't really understand that terribly well.

At some point we had to change some of these because I hadn't invented `Tournament::binary()` yet. I may not have reverted everything cleanly after I brought that in, however – sorry. I'll try to clean that up.